source/mainnet/net/resources/release-notes-mainnet.rst

```diff
@@ -15,6 +27,13 @@ Concordium Desktop Wallet 1.4.2 is a hotfix release specifically for macOS conta
 -   Fixed an issue on macOS where an error popup would show after closing the main application window and opening it again.
 -   Fixed an issue on macOS where identity creation was not possible after closing the main application window and opening it again.
 
+July 4, 2022
+
+Concordium Node 4.2.1
+---------------------
+
+Node startup has been improved and memory consumption reduced.
+
```

Comment:
I'm confused by this change. Isn't there already a section for the July 4 release above?

Reply:
It was because the desktop wallet release happened and got merged. It's fixed now.